client/my-sites/plans/jetpack-plans/product-store/pricing-banner/index.tsx

```diff
@@ -7,9 +9,19 @@ import './style.scss';
 export const PricingBanner: React.FC< ProductSlugsProps > = ( { siteId, duration } ) => {
 	const productSlugs = useProductSlugs( { siteId, duration } );
 
+	// TO-DO: Once Jetpack Social pricing tiers and the introductory discount have been worked on, filtering should be removed.
+	const filteredProductSlugs = useMemo(
+		() =>
+			productSlugs.filter(
+				( productSlug ) =>
+					! ( JETPACK_SOCIAL_PRODUCTS as ReadonlyArray< string > ).includes( productSlug )
+			),
```

Comment:
Maybe move this call to `.filter()` to `useProductSlugs` to avoid double memoization?

Reply:
Hi @manzoorwanijk Now that I think about it.. useProductSlugs isn't being used by other components so yes I agree it is better I just add it directly from there.